akka-actor-testkit-typed/src/main/resources/reference.conf

```diff
@@ -28,4 +28,8 @@ akka.actor.testkit.typed {
   # Throw an exception on shutdown if the timeout is hit, if false an error is printed to stdout instead.
   throw-on-shutdown-timeout=true
 
+  # Duration of LoggingEventFilter.intercept waits after the block is finished until
```

Comment:
Maybe clearer?
```
  # Duration to wait in LoggingEventFilter.intercept after the block is finished and
  # all required messages are received. Dilated by the timefactor.
```

Reply:
thanks, wasn't completely correct with how it works but I have changed to something that should be more clear